src/components/RoomView/clue/Clue.tsx

```diff
@@ -142,13 +142,22 @@ const Clue = () => {
           className={
             imRedSpy ? 'clue-submit-button dark-red-background-color' : 'clue-submit-button dark-blue-background-color'
           }
+          type={'submit'}
           onClick={handleSubmit}
+          onKeyUp={(e) => {handleSubmit(e) } }
```

Comment:
onKeyUp={(e) => { e.keyCode===13 && handleSubmit(e) } }

e.keycode is no longer in use / deprecated and we didn't actually need this here at all. just -->  onKeyUp={(e) => {handleSubmit(e) } }